src/effects/native/reverbeffect.h

```diff
@@ -17,23 +17,8 @@
 #include "util/types.h"
 
 struct ReverbGroupState {
-    ReverbGroupState() {
-        // Default damping value.
-        prev_bandwidth = 0.5;
-        prev_damping = 0.5;
-        reverb.init();
-        reverb.activate();
-        crossfade_buffer = SampleUtil::alloc(MAX_BUFFER_LEN);
-    }
-
-    ~ReverbGroupState() {
-        SampleUtil::free(crossfade_buffer);
-    }
-
+    float sampleRate  = 0;
```

Comment:
it's usually considered bad practice to not initialize all variables in the constructor